advocacy_docs/supported-open-source/patroni/index.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Patroni'
+description: 'Introduction to Patroni, a template to create your own high-availability solution'
+tags:
+  - topology
+  - Patroni
+indexCards: none
+directoryDefaults:
+  iconName: 'PostgresSupport'
+  product: 'Patroni'
+navigation:
+- platform_support
+- "#Installing"
+- installing_on_rhel
+- installing_on_debian
+- installing_etcd
+- "#Configuring"
+- configure_patroni
+- configure_haproxy
+- "#Using"
+- connecting
+- cluster_management
+---
+
+Patroni is among the most popular open-source tools for managing replication and failover of Postgres clusters. It relies on Postgres’s streaming replication and hot standby capabilities allowing DBAs to simplify the process of setting up and managing clusters having high availability as well as read scalability requirements. Patroni is tested by EDB for use with PostgreSQL and EDB Postgres Advanced Server.
+
+!!! Note
+    Looking for Patroni documentation? Head over to [https://patroni.readthedocs.io](https://patroni.readthedocs.io/en/latest/)
+
+## Supported topology
+
+Patroni supports these high availability components:
+* Postgres cluster: the database cluster, usually consisting of a primary and two or more replicas
+* Patroni: used as the failover management utility
+* etcd: used as a distributed configuration store (DCS), containing cluster information such as configuration, health, and current status
```

Comment:
It might be good to have a bullet item here for HAProxy since it's in the diagram just below this.

Reply:
Adding that as well now